libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/provider/MapboxNavigationProvider.kt

```diff
@@ -0,0 +1,65 @@
+package com.mapbox.services.android.navigation.v5.provider
+
+import android.content.ContentProvider
+import android.content.ContentValues
+import android.content.Context
+import android.content.pm.ProviderInfo
+import android.database.Cursor
+import android.net.Uri
+import com.mapbox.services.android.navigation.v5.internal.accounts.Billing
+import com.mapbox.services.android.navigation.v5.internal.accounts.MapboxNavigationAccounts
+import timber.log.Timber
+
+class MapboxNavigationProvider : ContentProvider() {
+
+    private val TAG = "MbxNavInitProvider"
```

Comment:
Same as above re: constants under `companion object`s

Reply:
What about calling the `TAG` `MapboxNavigationProvider` instead (no abbreviations)?

Reply:
Done